mobile/ios/Podfile.lock

```diff
@@ -149,6 +149,6 @@ SPEC CHECKSUMS:
   social_share: 702a5e3842addd22db515aa9e1e00a4b80a0296d
   url_launcher_ios: 08a3dfac5fb39e8759aeb0abbd5d9480f30fc8b4
 
-PODFILE CHECKSUM: cc1f88378b4bfcf93a6ce00d2c587857c6008d3b
+PODFILE CHECKSUM: 224e4680f2693f5d23f912e9aea395f4474ec34c
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.12.1
```

Comment:
Please don't commit the changes in this file.

Reply:
Will avoid this from next time.